manifesto.md

```diff
@@ -0,0 +1,22 @@
+# Draft: Sustainable Software Engineering Manifesto
+## We, the undersigned, are striving for resource-efficient software engineering practices, including the whole life cycle of software: Design, Development, Deployment, Operation, Shutdown. Through this work, we have come to value ...
```

Comment:
I think it is 'signatories', instead of 'undersigned'

Reply:
The undersigned passt: https://dictionary.cambridge.org/de/worterbuch/englisch/undersigned